ios/gutenberg.xcodeproj/xcshareddata/xcschemes/gutenberg.xcscheme

```diff
@@ -36,10 +36,10 @@
          </BuildActionEntry>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "YES"
+            buildForRunning = "NO"
```

Comment:
No sure what those mean, are we disabling the some types of build?

Reply:
Umm, this should not be necessary... No we're not disabling anything additional for this, this can be removed

Reply:
Actually my bad, this is necessary when building the app for the simulator, without this it leads to an architecture conflict when building the app